backend/store/cache/cache.go

```diff
@@ -176,15 +176,15 @@ func (r *Resource) Get(namespace string) []Value {
 	return r.cache[namespace]
 }
 
-// GetAll returns all cached resources across all namespaces.
-func (r *Resource) GetAll() []Value {
-	values := []Value{}
+// Size returns the size of all cached resources across all namespaces.
+func (r *Resource) Size() int {
+	var size int
```

Comment:
Let's create a new method instead of replacing GetAll. It is likely still useful for other purposes, and it would be preferable to not remove an exported method.

Reply:
We're not using it anywhere 😞as it was specifically implemented for the purpose of getting the total count. I can add it back though if we think there's value there (🤞the linter doesn't complain).

Reply:
Linter will only complain about unexported unused functions, as it doesn't know what is importing the package.